_rules/html-page-title-descriptive-c4a8a4.md

```diff
@@ -48,7 +48,9 @@ The target element describes the topic or purpose of the overall content of the
 
 ## Assumptions
 
-This rule assumes that the language of each test target can be correctly determined (either programmatically or by analyzing the content), and sufficiently understood.
+- This rule assumes that the language of each test target can be correctly determined (either programmatically or by analyzing the content), and sufficiently understood.
+- This rule is testing Technique G88: Providing descriptive titles for Web pages
```

Comment:
```suggestion
```

Reply:
Don't think this one is an assumption, and, since it is covered in the Bibliography, I suggest we just remove it from here.